### ISSUE TITLE
Publishing from master instead of main

### ISSUE THREAD

**Issue by abachma2** [cyclus/cyclus.github.com#347]
With the changes in #345 and #346 it looks like the website is still published from the master branch and not the main branch like it should be. This may be why some of the most recent changes are not captured in the current version of the site.

**Comment by gonuke** [comment]
I think it's something different from that.

1. I didn't rebuild/republish to main
2. There is another PR that includes more comprehensive updates from master -> main

**Comment by abachma2** [comment]
OK. I thought that had already been merged. Merging #341 will close this issue then.

**Comment by gonuke** [comment]
#341 is now merged, but someone needs to rebuild/publish the website to see final updates

**Comment by abachma2** [comment]
I tried to build it, it all runs fine, but I'm not sure it actually includes all of the updates made in #335.

**Comment by gonuke** [comment]
It might be because of errors in the build?  I noticed that `cep0` doesn't render correctly with the new CEP 29 showing up in the directory

**Comment by abachma2** [comment]
Maybe, because the changes from #340 are present.

**Comment by bennibbelink** [comment]
Is this issue still relevant after #349 was merged?

**Comment by abachma2** [comment]
Let me try building the website from the source branch and see what happens.

**Comment by abachma2** [comment]
Based on the terminal output, all the build commands are pointing to the `main` branch. The main point of this issue is resolved, so I'll close the issue.
